Example/StripeConnectExample/StripeConnectExampleTests/StripeConnectExampleTests.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  StripeConnectExampleTests.swift
+//  StripeConnectExampleTests
+//
+//  Created by Chris Mays on 8/21/24.
+//
+
+import XCTest
+@testable import StripeConnectExample
+
+final class StripeConnectExampleTests: XCTestCase {
```

Comment:
I don't think we need to write any unit tests against the example app, but rather just E2E UI tests. So we can likely delete this file and the corresponding test target.

But we should keep the UITests target.